master/buildbot/newsfragments/setuptools-is-required.bugfix

```diff
@@ -0,0 +1 @@
+setuptools is now marked as required to install buildbot (non-setuptools install was not tested and broken).

```

Comment:
@tomprince I don't see the use of logging .misc newsfragments (git log is fine for that). But this fix is actually imo useful to heads up to users